### DISCUSSION TITLE
Trailing stop

### DISCUSSION THREAD

**Discussion by Aalv3** [MicroTrendsLtd/NinjaTrader8#83] [General]
Where can I change the trail stop settings to off? I've managed to integrate the trade panel into my strategies, but can't figure out how to stop it from trailing my stop

**Comment by MicroTrendsTom** [comment]
read inside the code of the samples to see  for trade management section 
edit it and add your own
eg:
https://github.com/MicroTrendsLtd/NinjaTrader8/blob/main/ATSQuadroStrategyBase/NinjaTrader%208/bin/Custom/Strategies/ATSSamplePriceReversalTestHybridAlgo.cs


this property: IsTradeManagementEnabled
should allow automatic on off   or you need to add inline  externally prior to the  method call  to TradeManagement(..)


public override void TradeManagement(double lastPrice)
{

}

**Reply by MicroTrendsTom** [reply]
base class had some fixes - pls check out the latest code base - beware folder name changes

**Reply by Aalv3** [reply]
Have any notes on what the changes are? 4000+ lines of code to read and compare is a lot lol

**Reply by MicroTrendsTom** [reply]
check the changes out etc
https://github.com/MicroTrendsLtd/NinjaTrader8/commit/af919b27c31d3e31b52c0f3cc675095c14bc56e2

**Reply by MicroTrendsTom** [reply]
you can browse them here:  its the same method in all git projects repos etc
https://github.com/MicroTrendsLtd/NinjaTrader8/commits/main

then you can see notes added by the dev and the changes

**Comment by josecali** [comment]
Hi @Aalv3
Were you able to disable the trailing stop setting?
Can you tell me how you solved it?
Thanks.
